static/app/views/organizationStats/usageChart/index.tsx

```diff
@@ -38,9 +38,10 @@ const COLOR_TRANSACTIONS = Color(commonTheme.dataCategory.transactions)
 const COLOR_ATTACHMENTS = Color(commonTheme.dataCategory.attachments)
   .lighten(0.65)
   .string();
-
-const COLOR_DROPPED = commonTheme.red300;
-const COLOR_FILTERED = commonTheme.pink100;
+const COLOR_FILTERED = commonTheme.purple200;
+const COLOR_RATE_LIMITED = commonTheme.green200;
+const COLOR_INVALID = commonTheme.red300;
+const COLOR_CLIENT_DISCARD = commonTheme.yellow300;
```

Comment:
@Jesse-Box How do you feel about these colors?

Reply:
Green is a color usually associated to success, but rate limits drop events. I'd use green (or blue) rather for accepted and red, yellow, orange etc for all the other outcomes.

Reply:
I would love to use orange, but apparently, we do not have this color in the theme